assets/javascripts/admintable.js

```diff
@@ -40,7 +46,9 @@ function addAdminTableRow() {
     // add new row
     var newRow = adminTable.row.add(adminTable.emptyRow);
     var newRowIndex = newRow.index();
-    adminTable.rowData[newRowIndex] = jQuery.extend({isEditing: true}, adminTable.emptyRow);
+    adminTable.rowData[newRowIndex] = jQuery.extend({
+        isEditing: true
+    }, adminTable.emptyRow);
```

Comment:
The previous style looked better.

Reply:
see https://github.com/os-autoinst/openQA/pull/3268#discussion_r456940506